README.rst

```diff
@@ -37,12 +37,19 @@ For Travis CI
 
 1. Activate CI for your github repository on `Travis CI <https://travis-ci.org/>`_). You may do so either at https://travis-ci.org/profile/YOUR_GITHUB_ORGANIZATION or at https://travis-ci.org/profile/YOUR_GITHUB_USER (depending on where your repository sits).
 
-2. In `.travis.yml` file in your client repo, add in "`install`" section a sentence `git clone https://github.com/ros-industrial/industrial_ci.git .industrial_ci`, like below:
+2. Add `.travis.yml` file to your repository root (`complete template <https://github.com/ros-industrial/industrial_ci/blob/master/doc/.travis.yml>`_):
 
 ::
+  language: generic
+  services:
+    - docker
+
+  env:
+    matrix:
+      - ROS_DISTRO="indigo"
```

Comment:
This portion doesn't seem to render as code block.
![screenshot from 2018-11-30 03-36-31](https://user-images.githubusercontent.com/1840401/49287137-3ac02780-f451-11e8-8d09-49b72b0804d5.png)

Reply:
I have removed a mandatory new line.. fixed